### ISSUE TITLE
must_not is not work and return the opposite result

### ISSUE THREAD

**Issue by Muniuliuma** [milvus-io/pymilvus#352]
I just try the demo，and use dsl to search ,below is my dsl
```
dsl = {
    "bool": {
        "must": [
            {
                "must": [
                    {
                        "vector": {
                            "embedding": {
                                "topk": 3,
                                "query": [filmA.get("embedding")],
                                "metric_type": "L2"
                            }
                        }
                    }
                ]
            },
            {
                "must":[
                    {
                        "must_not":[
                            {
                                "term":{
                                    "release_year":[2002,2001]
                                }
                            }
                        ]
                    }
                ]
            }
        ]
    }
}
results=client.search(collection_name,dsl,fields=["embedding","release_year"])
```
and results contains two entity which release_year is 2002 and 2001
Shouldn't is just return the results which release_year is not in [2001,2002]?
Is there is something wrong with my dsl or just it's a bug

**Comment by BossZou** [comment]
Thanks for issuing the bug. We'll fix it at next version.

**Comment by wineternity** [comment]
How to fix this bug? the pymilvus 0.4.0 is coresponding to milvus 0.11.1 which is not released
